src/js/playlist/playlist.js

```diff
@@ -48,9 +49,12 @@ define([
             if (androidhls) {
                 originalSource.androidhls =  androidhls;
             }
-
-            if (originalSource.protection || itemProtection) {
-                originalSource.protection = originalSource.protection || itemProtection;
+            if(_hasDrm()) {
```

Comment:
Edition enforcement should be done in commercial where the feature is is implemented, not here in os playlist source normalization.